main.py

```diff
@@ -1,7 +1,32 @@
 import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
 
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    items = ["Western", "Action", "Dramma"]
+    for item in items:
+        Genre.objects.create(name=item)
+    items = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    for first, last in items:
+        Actor.objects.create(first_name=first, last_name=last)
```

Comment:
In this case, you can use "name" and "surname" name instead of "first" and "last".